gpflow/kernels.py

```diff
@@ -60,7 +60,31 @@ def __init__(self, input_dim, active_dims=None, name=None):
             self.active_dims = np.array(active_dims, dtype=np.int32)
             assert len(active_dims) == input_dim
 
-        self.num_gauss_hermite_points = 20
+    def _validate_ard_shape(self, name, value, ARD=None):
```

Comment:
What does `name` mean?

Reply:
the name of the parameter, for sensible error messages

Reply:
wouldn't it better to use kwargs then and restrict kwargs to one value? like:
```
def _validate_ard_shape(self, ARD: Optional[bool] = None, **kwargs: Dict[str, np.ndarray])
```

Reply:
in any case, `name` doesn't have docstring